include/tudat/simulation/environment_setup/createGravityField.h

```diff
@@ -545,6 +687,30 @@ inline std::shared_ptr< GravityFieldSettings > sphericalHarmonicsGravitySettings
             );
 }
 
+inline std::shared_ptr< GravityFieldSettings > polyhedronGravitySettings(
+        const double gravitationalConstant,
+        const double density,
+        const Eigen::MatrixXd verticesCoordinates,
+        const Eigen::MatrixXi verticesDefiningEachFacet,
+        const std::string& associatedReferenceFrame)
+{
+    return std::make_shared< PolyhedronGravityFieldSettings >(
+            gravitationalConstant, density, verticesCoordinates,
+            verticesDefiningEachFacet, associatedReferenceFrame);
+}
+
+inline std::shared_ptr< GravityFieldSettings > polyhedronGravitySettings(
+        const double gravitationalParameter,
+        const Eigen::MatrixXd verticesCoordinates,
+        const Eigen::MatrixXi verticesDefiningEachFacet,
+        const std::string& associatedReferenceFrame,
+        const double density = TUDAT_NAN )
```

Comment:
Why is the density a separate input here? From the gravitational parameter, the volume (which is defined by the shape) and a gravitaional constant (to be provided as input, bu with a default), the density can always be calculated (?)

Reply:
I've modified this, so that the gravitational constant is an input, and the density gets calculated. In the settings object, I've also added the relevant recalculations in a reset of density or mu

Reply:
Sounds good.
I remember there was some reason why I used the density instead of the gravitational constant when implementing it, but right now the new interface seems much more logic